oak_runtime/tests/integration_test.rs

```diff
@@ -37,10 +37,26 @@ mod common {
     use wat::parse_str;
 
     pub fn start_runtime() -> Result<Arc<Runtime>, OakError> {
+        // Loop 100 000 times in the main function to make sure the Wasm node is alive for a while
+        // before exiting.
```

Comment:
Perhaps also explain that this is needed to avoid race conditions with metrics checks.

Reply:
Thanks, done.